hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/mode/ModeService.java

```diff
@@ -0,0 +1,45 @@
+package pl.allegro.tech.hermes.management.domain.mode;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class ModeService {
+    public enum ManagementMode {
+        READ_WRITE("readWrite"), READ_ONLY("readOnly");
+
+        private final String text;
+
+        ManagementMode(String text) {
+            this.text = text;
+        }
+
+        public String getText() {
+            return text;
+        }
+    }
+
+    private volatile ManagementMode mode = ManagementMode.READ_WRITE;
```

Comment:
We would like to set this flag in each hermes-management instance independently?

Reply:
Yes, it is the easiest option to realise and it should be sufficient for us now. In the near future we will work on more sophisticated mechanism such as auto switching from `READ_WRITE` to `READ_ONLY` when zookeeper is down.